espnet2/tts/gradtts/base.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2021. Huawei Technologies Co., Ltd. All rights reserved.
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# MIT License for more details.
+
+import numpy as np
+import torch
+
+
+class BaseModule(torch.nn.Module):
```

Comment:
We do not necessarily need this (as all the functions will be taken care during our espnet trainer). My suggestion would to simple use torch.nn.Module as base class.